_algolia.yml

```
@@ -2,7 +2,7 @@
 algolia:
     application_id: '5IGTHBX5JS'
     index_name:     'blog_eleven'
-    excluded_files:
+    extensions_to_index:
```

```suggestion
    files_to_exclude:
```

based on migration guide https://community.algolia.com/jekyll-algolia/migration-guide.html#options

> excluded_files has been renamed to files_to_exclude